CHANGELOG.md

```diff
@@ -1,7 +1,28 @@
 # CHANGELOG
 Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
-## [Unreleased]
+## [Unreleased 3.0]
```

Comment:
This section should not be here, the change should go to ## [Unreleased 2.x] / Added

Reply:
My bad, let me fix this.

Reply:
@reta I think this looks good now.